docs/sphinx_docs/Usage/index.md

```diff
@@ -26,6 +26,22 @@ Alt-Ergo supports file extensions:
 
 See the [Input section] for more information about the format of the input files
 
+### Generating models
+Since 2.5.0, Alt-Ergo also generates models in the case it concludes on the satisfiability of
+the formula.
+There is two ways to activate model generation:
+- `with the --interpretation=VALUE`, where VALUE can be equal to:
+  * "none", and alt-ergo will not generate models (by default);
+  * "first", and alt-ergo will output the first model it finds;
+  * "every", alt alt-ergo will compute a model before each decision
+  * "last", and alt-ergo will output the last model it computes before returning 'unknown'.
+  Note that this mode only works with the option `--sat-solver tableaux`.
+
+- with the `--model option`, setting the interpretation option to 'last', and the sat-solver
+to 'tableaux.
```

Comment:
maybe put this option first, as this is the recommended way to generate models (basically the `--interpretation` option is for advanced users/devs)